packages/components/ng-ovh-contacts/package.json

```diff
@@ -52,8 +52,8 @@
     "flag-icon-css": "^3.2.1",
     "moment": "^2.20.1",
     "ovh-api-services": "^9.0.0",
-    "ovh-ui-angular": "^2.23.x",
-    "ovh-ui-kit": "^2.23.x",
+    "ovh-ui-angular": "^3.16.0",
+    "ovh-ui-kit": "^2.41.1",
```

Comment:
I think this was already done on `develop` branch

Reply:
That's not really what was saying the `yarn.log` on my side when I've started it from `develop`. But it's not a bad idea to correct all of this. There was some `~` and fixed versions in some `package.json` :)